src/stories/Terminal.js

```diff
@@ -83,12 +83,7 @@ class InteractiveTerm extends Terminal {
         const printable =
           !ev.altKey && !ev.altGraphKey && !ev.ctrlKey && !ev.metaKey
         // Ignore arrow keys
-        if (
-          ev.code === 'ArrowUp' ||
-          ev.code === 'ArrowDown' ||
-          ev.code === 'ArrowLeft' ||
-          ev.code === 'ArrowRight'
-        ) {
+        if (ev.code === 'ArrowUp' || ev.code === 'ArrowDown') {
```

Comment:
It seems I cannot use `backspace` in the current storybook, can you check that as well?

Reply:
@sradevski  Yea that was how it was already and I'm not sure how to fix it.
I guess that it has to do with the fact that we are using a dummy input, instead of a tty which responds back.
Listening to the Backspace button isn't hard, but now that we can move the cursor, it felt like more work that what I was planning for the needs of this bugfix PR.

Reply:
@thgreasi Yeah you can leave it as it is, not a big deal. Also there is [this bug](https://github.com/balena-io/resin-ui/issues/2682) that we need to resolve ASAP, but hopefully it will be resolved upstream soon.